_posts/2017-07-05-puppet-ruby-facts.md

```diff
@@ -352,16 +352,16 @@ and this would output a single number, instead of a string:
 
 ### Structured Facts
 
-[Structured facts](https://docs.puppet.com/facter/3.7/custom_facts.html#structured-facts) allow people to send more data than just a simple text string.  This is usually as encoded JSON or YAML data.  External facts have been able to provide structured facts e.g. A batch file that outputs pre-formatted JSON text, however this is not enabled for PowerShell yet (https://tickets.puppetlabs.com/browse/FACT-1653).
+[Structured facts](https://docs.puppet.com/facter/3.7/custom_facts.html#structured-facts) allow people to send more data than just a simple text string.  This is usually as encoded JSON or YAML data.  External facts have been able to provide structured facts, for instance using a batch file to output pre-formatted JSON text, but this is not yet enabled for PowerShell (https://tickets.puppetlabs.com/browse/FACT-1653).
 
 ### `puppet facts` vs `facter`
```

Comment:
I think this  probably be clarified a bit more... keep in mind there is `facter`, `puppet facts` and `facter -p` which isn't mentioned here.